.rgignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
Two thoughts:
1) If we have `.rgignore` should we add `.agignore` and `.ackrc` for consistency?
2) I've been tripped up before by my own ignore configurations, scratching my head while I can't find something only to stumble across it when I give up and `grep` and realize my mistake. I'm all for ignoring generated files -- in fact, most of these tools already honour `.gitignore` by default -- but I'm worried it will obscure `vendor` for newcomers trying to find code.

~1.999.../5

Reply:
I'd lean towards removing it and discussing in another PR since this is out-of-scope, only 1/5 though

Reply:
Ok I dropped it because it's out of scope.